docs/_layouts/default.html

```diff
@@ -59,7 +59,7 @@
         </div>
 
         <!-- Concierge button anchored at the bottom of the LHN (rendered on wider screens) -->
-        {% include floating-concierge-button.html id="floating-concierge-button-lhn" %}
+        <!-- {% include floating-concierge-button.html id="floating-concierge-button-lhn" %} -->
```

Comment:
I think it's probably safe to just get rid of this. If we need to get it back, we can just reference this PR!

Reply:
And you could probably find `floating-concierge-button.html` and kill that file off too